modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1904,6 +1908,43 @@ public void testGetObjectEntriesAggregationFacets() throws Exception {
 		Assert.assertEquals(facetValue.getNumberOfOccurrences(), (Integer)2);
 	}
 
+	@Test
+	public void testGetObjectEntriesWithAggregationFieldAndManyToManyRelationshipWithHimself()
```

Comment:
```suggestion
	public void testGetObjectEntriesAggregationManyToManyRelationship()
```
If you see, methods names don't have prepositions on their names. Just a matter of patterns.

Just `Relationship` at the end of the method name looks more general, if we want to add some other tests relating to relationships, they could be developed inside this new method.

Remember to order the methods alphabetically after the change.